src/GovUk.Education.ExploreEducationStatistics.Admin/Services/PublicationService.cs

```diff
@@ -148,12 +148,29 @@ public class PublicationService : IPublicationService
 
                     return (await ValidatePublicationSlugUniqueForUpdate(publication.Id, updatedPublication.Slug)).Map(_ =>
                     {
+                        var statisticsDbPublication = _statisticsDbContext.Publication.Find(publication.Id);
+                        if (statisticsDbPublication != null)
+                        {
+                            _statisticsDbContext.Update(statisticsDbPublication);
+                            statisticsDbPublication.Slug = updatedPublication.Slug;
+                            _statisticsDbContext.SaveChanges();
+                        }
+
                         publication.Slug = updatedPublication.Slug;
                         return publication;
                     });
                 })
                 .OnSuccess(async publication =>
-                {
+                { 
+                    var statisticsDbPublication = await _statisticsDbContext.Publication.FindAsync(publication.Id);
+                    if (statisticsDbPublication != null)
+                    {
+                        _statisticsDbContext.Update(statisticsDbPublication);
+                        statisticsDbPublication.Title = updatedPublication.Title;
+                        statisticsDbPublication.TopicId = updatedPublication.TopicId;
```

Comment:
Is this topic guaranteed to exist at the point? What happens if no subjects got imported for it yet?